.github/workflows/test-csharp.yml

```diff
@@ -63,6 +64,14 @@ jobs:
 
       - run: echo '${{ steps.stepid.outputs.dotnet-version }}'
 
+      - name: Set VersionSuffix environment variable
+        if: github.event_name == 'workflow_dispatch' && startswith(matrix.os, 'ubuntu')
+        run: echo "VersionSuffix=manualtrigger" >> $GITHUB_ENV'
+
+      - name: Set VersionSuffix environment variable
+        if: github.event_name == 'workflow_dispatch' && startswith(matrix.os, 'windows')
```

Comment:
Is this needed? `The enclosing job build_nuget` runs only on 'ubuntu_latest'

Reply:
Yeah, it is needed for the `test_nuget` job. I originally was going to set both again inside that job, but found that env variables seem to be preserved across steps so I put them both here. If you think it makes more sense I can still add them to the `test_nuget` job.

Reply:
Yeah that works, thanks!